docs/build-guides/mac-build.md

```diff
@@ -8,7 +8,7 @@ Automated build: clone the repo and run `mac.build.sh` in the `libindy` folder.
 2. Install required native libraries and utilities (libsodium is added with URL to homebrew since version<1.0.15 is required)
```

Comment:
With the change in this PR, we should remove this piece of doc
>  (libsodium is added with URL to homebrew since version<1.0.15 is required)
as well to avoid confusion